### ISSUE TITLE
Buggy URL filter

### ISSUE THREAD

**Issue by cjwyatt** [dalaranwow/dalaran-wow#5110]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Trying to send the message, "oh lol, you're 64 not 74, my bad" to a friend in whisper causes the message to blocked by the URL filter.
**Expected behaviour**: Tell us what should happen instead
It's clearly not a URL so it should be allowed.
**Steps to reproduce the problem**:

1. Send a private message to someone saying, "oh lol, you're 64 not 74, my bad"
2. Observe it is incorrectly filtered.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by StheNoble** [comment]
yea, nearly any message with commas "," get filtered. goldspammers braught that uppon us. 
Those filters are pretty pointless though since spammers just adept within an our but the rest of the peopel have to deal with those filters wich even stop many messages with just a linked item wich is anoying when loot counceling and such.

**Comment by TeridanDW** [comment]
@Lothloryen

**Comment by Aridam** [comment]
Confirmed from my side as well -- same applies to those annoying goldseller spam whispers. Seems it has no practical value at this point.

**Comment by KerDW** [comment]
Can confirm, the goldsellers always found some way to bypass the filter anyway, it should just be removed IMO.

**Comment by cjwyatt** [comment]
![image](https://user-images.githubusercontent.com/6775797/33239328-82efecf2-d297-11e7-87fa-eda1c8996a9e.png)

As an example of how this only affects players whilst gold sellers just work around it.

**Comment by datbunneh** [comment]
E.g. Ring of Rapid Ascent heroic (50664) cannot be linked.

**Comment by Lothloryen** [comment]
I will perform some improvments, thank you.

**Comment by Aridam** [comment]
Still an issue -- goldsellers still at large and bypassing this, meanwhile some Items cannot be linked, and some messages/macros/scripts get treated as URLs despite not being that and get blocked as such.
